corehq/apps/sms/models.py

```diff
@@ -177,6 +177,7 @@ class SMSBase(UUIDGeneratorMixin, Log):
     ERROR_PHONE_NUMBER_OPTED_OUT = 'PHONE_NUMBER_OPTED_OUT'
     ERROR_INVALID_DESTINATION_NUMBER = 'INVALID_DESTINATION_NUMBER'
     ERROR_MESSAGE_TOO_LONG = 'MESSAGE_TOO_LONG'
+    ERROR_MESSAGE_BLANK = 'MESSAGE_BLANK'
```

Comment:
`ERROR_NO_MESSAGE` looks like it has the same purpose: https://github.com/dimagi/commcare-hq/blob/567f7256acb30e71674f4fb1e3be66ff25dab477/corehq/messaging/scheduling/models/content.py#L48-L50

Reply:
Is is the same purpose but not all messages go through this workflow, specifically sms surveys: https://github.com/dimagi/commcare-hq/blob/dd7129005ba038441a53e6a7c1951c0ce29e4d14/corehq/apps/smsforms/tasks.py#L56-L79

What I will do though is update the code linked above to apply the same error status to the subevent (2e65d67db37) but I'm going to leave the SMS code as a fallback.

Reply:
Is there value to keeping the two different error constants? They seem like they mean the same thing, just in different workflows.

Reply:
I think there is value in keeping them separate since they are on different models and the UI text would be different (more generic for the SMS error than for the subevent error.

I actually wonder if I shouldn't create a new message for this one since the UI text for the other one doesn't really make sense in the context of sms surveys: "No message available for the given language settings."

Reply:
Ah, I hadn't looked up the message text for `ERROR_NO_MESSAGE`, I was just going based on the constant name.